### DISCUSSION TITLE
How to load dollyv2 model which has already been downloaded and kept for Named Entity Recognition

### DISCUSSION THREAD

**Discussion by yazeedp** [explosion/spacy-llm#441] [Q&A]
I already have dolly model which has been downloaded from hugging face and kept on my local drive. Can you assist on if there is a way to provide path to a model kept on hard disk drive in the code?

**Comment by svlandeg** [comment]
Hi! 

You can set an environment variable `HF_HOME` on your system, to point to the HuggingFace cache. For instance, on my system this folder has a subfolder "hub" which has another subfolder "models--databricks--dolly-v2-12b". Whenever then the code would use HF's Dolly v2 12b model, it will access it from that location.

**Reply by yazeedp** [reply]
I tried changing the HF_HOME env variable to point to the local model path but still it is not working. The cached version of the model when downloaded while the code is being run and auto downloaded from the hub contains other folders like blobs, refs, snapshots and .no_exist subfolders etc. But on downloading the model from hugging face directly, the instruct pipeline config.json, model_bin.py etc are there. The sub folders structure present in the cache are not there. So i am not able to load the model
